packages/@aws-cdk/aws-apigateway/test/integ.api-definition.inline.private.ts

```diff
@@ -0,0 +1,67 @@
+import * as cdk from '@aws-cdk/core';
+import * as apigateway from '../lib';
+
+/*
+ * Stack verification steps:
+ * * `curl -i <CFN output PetsURL>` should return HTTP code 200
```

Comment:
Can you add verification steps that confirm that the Rest API is in fact configured behind a private endpoint.

Have you verified that this works by running `yarn integ integ.api-definition.inline.private.js`? 

See https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#integration-tests

Reply:
I added documentation about how this works in 44618d9. If you got the private DNS enabled it should just work as if it was public (if you network if configured accordingly. I added also a link to the documentation where indicates how to invoke a private API.

I am happy to remove the integration test if you think this could be more confusing that useful.